wrappers/wrapper.rb

```diff
@@ -372,6 +386,10 @@ def assert_no_vehicle_free_approach_or_return_if_heuristic(vrp)
       vrp.vehicles.none?{ |vehicle| vehicle[:free_approach] || vehicle[:free_return] } || vrp.preprocessing_first_solution_strategy.to_a.first != 'periodic'
     end
 
+    def assert_no_free_approach_or_return(vrp)
+      vrp.vehicles.none?{ |vehicle| vehicle[:free_approach] || vehicle[:free_return] }
```

Comment:
```suggestion
      vrp.vehicles.none?{ |vehicle| vehicle.free_approach || vehicle.free_return }
```

Reply:
@braktar  there are others just above as well. While you are at it, you can do them all maybe?